docs/guides/specify-runtime-options.mdx

```diff
@@ -0,0 +1,403 @@
+---
+title: Specify options
+description: Specify options when building with Qiskit Runtime primitives
+
+---
+
+# Specify options
+
+You can use options to customize the Estimator and Sampler primitives. This section focuses on how to specify Qiskit Runtime primitive options. While the interface of the primitives' `run()`  method is common across all implementations, their options are not. Consult the corresponding API references for information about the [`qiskit.primitives`](/api/qiskit/primitives#primitives) and [`qiskit_aer.primitives`](https://qiskit.github.io/qiskit-aer/apidocs/aer_primitives.html) options.
+
+Options are specified differently depending on whether you are using the V1 or V2 interface.
+
+## V2 primitives
+
+Options are specified differently in the V2 primitives in these ways:
+
+- `SamplerV2` and `EstimatorV2` have separate options classes. You can see the available options and update option values during or after primitive initialization.
+- Instead of the `set_options()` method, V2 primitive options have the `update()` method that applies changes to the `options` attribute.
+- If you do not specify a value for an option, it is given a special value of `Unset` and the server defaults are used.
+- For V2 primitives, the `options` attribute is the `dataclass` Python type.  You can use the built-in `asdict` method to convert it to a dictionary.
+
+## Instantiate the Options class (V1)
```

Comment:
This section is no longer needed since the following section, "Set primitive options", already talks about how to instantiate the options class for V1.